spec/echo-webhooks.yaml

```diff
@@ -532,7 +532,9 @@ tags:
             ````json
             {
                 "id": "f9d421d5-63e3-4724-8a21-d7d81d81417a",
+                "timestamp": "2019-04-23T08:46:12Z",
                 "type": "notification_batch.created",
+                "source": "system",
```

Comment:
This example doesn't really make sense, as timetable_change events won't be generated by system.

Reply:
What will generate it then?

Reply:
`notification_channel:<channel-name>`, dus bijvoorbeeld `notification_channel:webhook.example`.

Reply:
Not sure if we want to change the example now. In the new set up (examples being generated, as in `echo.yaml`), it will not be an issue anymore? It's just `system` for both `notification_batch.created` and `ping` types.